share/templates.d/99-generic/efi.tmpl

```diff
@@ -21,9 +21,6 @@ install boot/grub2/fonts/unicode.pf2 ${EFIBOOTDIR}/fonts/
 
 ## actually make the EFI images
 ${make_efiboot("images/efiboot.img")}
-%if domacboot:
-    ${make_efiboot("images/macboot.img", imgtype="apple")}
-%endif
 
```

Comment:
Why is this being removed?

Reply:
That is, without this, we don't get in the Mac EFI menu the fancy "Fedora" icon that makes it obvious what we are to the firmware when booting the stick.

Reply:
+1 to @Conan-Kudo 's question here. It doesn't seem necessary to remove the integration with the intel mac efi firmware.

Reply:
Because, at least in my testing, it isn't needed. And I like thinks to be as simple as possible. It's not too hard to tell what it is when booting, even without the logo.

Reply:
Does it work when the plist and logo files are installed in the regular ESP image?

Reply:
No idea, but I would guess not. IIRC plist neeed to be 'blessed' in order for it to be loaded. There are so many variations to all these things that I'm not going to explore all of them just to get a logo on there.

Reply:
Ahh, yeah, so `grub2-macbless` only works with HFS+ ESP, rather than FAT32 ESP.

Reply:
I'm not actually sure if removing this will break on my 2009 MacBook Pro (which I use as a Linux laptop), I will need to test this... Last I tried it only showed me stuff with the HFS+ ESP...

Reply:
Yeah, confirmed, my 2009 MBP isn't super-happy with the FAT ESP...